rlog_generator/utils.py

```diff
@@ -42,6 +40,8 @@ def load_config(yaml_file):
             if 'locale' in config and config['locale']:
                 global fake
                 fake = Faker(config['locale'])
+            else:
+                fake = Faker()
```

Comment:
Allows faker to be initialized if there is no locale set, otherwise the program should most likely crash?